pkg/api/v1/types.go

```diff
@@ -2520,6 +2520,17 @@ type PodSpec struct {
 	// +patchMergeKey=IP
 	// +patchStrategy=merge
 	HostAliases []HostAlias `json:"hostMappings,omitempty" patchStrategy:"merge" patchMergeKey:"IP" protobuf:"bytes,23,rep,name=hostMappings"`
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by a
+	// PriorityClass object with that name.
```

Comment:
different comments with `api/types.go`, should be `by creating a PriorityClass`.